cfg/conf.sample.php

```diff
@@ -1,4 +1,5 @@
-;<?php http_response_code(403); /*
+;<?php http_response_code(403); 
+/*
```

Comment:
If you want to move this PHP-comment down, you should prefix the line with a INI-comment so it doesn't get parsed. The purpose of the PHP-comment is to hide the remaining content from the PHP parser so it wont get returned if accessible from the web.

I would suggest keeping it on the one line, though.

Reply:
Done